OpenRA.Mods.Common/Widgets/DropDownButtonWidget.cs

```diff
@@ -41,14 +41,11 @@ public override void Draw()
 
 			var image = ChromeProvider.GetImage("scrollbar", IsDisabled() ? "down_pressed" : "down_arrow");
 			var rb = RenderBounds;
-			var color = GetColor();
-			var colorDisabled = GetColorDisabled();
 
 			WidgetUtils.DrawRGBA(image, stateOffset + new float2(rb.Right - rb.Height + 4, rb.Top + (rb.Height - image.Bounds.Height) / 2));
 
-			WidgetUtils.FillRectWithColor(new Rectangle(stateOffset.X + rb.Right - rb.Height,
-				stateOffset.Y + rb.Top + 3, 1, rb.Height - 6),
-				IsDisabled() ? colorDisabled : color);
+			var separator = ChromeProvider.GetImage("separator", "separator");
```

Comment:
How about "dropdown", "separator"?  "separator" by itself could be almost anything.